frontend/src/server/Room.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+var app = require("express")();
+var http = require("http").Server(app);
+var io = require("socket.io")(http);
+
+const port = 3000;
+
+app.get("/", function(req, res) {
+  res.sendFile(__dirname + "/index.html");
```

Comment:
Is this using react? I believe this is instead using express, which is a different framework.